testng-core/src/test/java/test/listeners/issue2771/TestCaseSample.java

```diff
@@ -0,0 +1,13 @@
+package test.listeners.issue2771;
+
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+@Listeners(CustomSoftAssert.class)
+public class TestCaseSample {
+  @Test
+  public void someCustomSoftAsserts() {
+    // doing custom soft asserts
```

Comment:
Something is missing or is it expected?

Reply:
No nothing is missing. Its just for illustrative purposes. A real test would be doing some custom operations. I have just borrowed the example from the issue thats all.